frontend/src/concepts/design/CollapsibleSection.tsx

```diff
@@ -46,28 +44,21 @@ const CollapsibleSection: React.FC<CollapsibleSectionProps> = ({
             aria-expanded={open}
             isInline
             variant="link"
```

Comment:
Is this an existing component within the product that we're reusing for this page?

When I review the latest implementation on the poc cluster, there are a few observations about the styles for this button:

### Button variant
Currently this button is use 
```
isInline
variant="link"
```
But for an icon button, the ideal variant is `variant="plain"` without the `isInline` modifier, similar to the icon button in this [PF react example](https://www.patternfly.org/components/button#variant-examples). 

### Spacing
I get why the `isInline` modifier is used to remove padding, and it makes sense to remove padding for this. If you remove that, the plain variant would add padding to the button. Another option for removing the left & right padding is to use a utility class `pf-v5-u-px-0`.

### Sizing
Ideally any icons that accompany text would have the same size as the text. You can do this by adding a utility class to match the size of the header. In this example, you would want it to be `pf-v5-u-font-size-2xl` but if the header size ever changes, then the size of this icon would need to be updated, too.

Reply:
We do not import the pf utility classes

Reply:
I will update as you suggest though but I'll use styles.